commit_messages/safe.txt

```diff
@@ -387,7 +360,6 @@ Updated framework to the lattest version
 Use a real JS construct, WTF knows why this works in chromium.
 Useful text
 Version control is awful
-WHO THE FUCK CAME UP WITH MAKE?
 WIP, always
 WIPTF
 WTF is this.
```

Comment:
There's some argument to be made that all of the `*TF` messages also belong in the unsafe file. They're not really usefully censorable, but they're not really _censored_ per se, either.